extra/resources/controld

```diff
@@ -228,7 +228,14 @@ controld_monitor() {
     killall -0 ${OCF_RESKEY_daemon} >/dev/null 2>&1 ; rc=$?
 
     case $rc in
-      0) rc=$OCF_SUCCESS;;
+      0) dlm_tool ls | grep -q "wait fencing"
```

Comment:
I cant find any other way to check the status.

"dlm_tool ls" returns 0 as long as DLM is up and running it seems (I tried when the status was in "wait quorum" and "wait fencing").